.ci-mgmt.yaml

```diff
@@ -10,3 +10,4 @@ plugins:
 team: ecosystem
 env:
   VAULT_DEV_ROOT_TOKEN_ID: "root"
+  VERSION_PREFIX: 6.0.0
```

Comment:
The `VERSION_PREFIX` env is unfortunately stateful. It needs to exist for the master PR to merge, but should not exist into the future (otherwise builds fail). The best way I have found to do this is to set `VERSION_PREFIX: 6.0.0` in `.ci-mgmt.yaml`, run `make ci-mgmt` and then unset `VERSION_PREFIX`. That way the next workflows run will clean up your workflows after the release.

```suggestion
```